pay-api/src/pay_api/services/fas/routing_slip.py

```diff
@@ -322,10 +322,11 @@ def create(cls, request_json: Dict[str, any], **kwargs):
             status=CfsAccountStatus.PENDING.value
         ).flush()
 
-        total = sum(float(payment.get('paidAmount')) for payment in request_json.get('payments'))
+        total = get_quantized(sum(float(payment.get('paidAmount')) for payment in request_json.get('payments')))
 
         # Calculate Total USD
-        total_usd = sum(float(payment.get('paidUsdAmount', 0)) for payment in request_json.get('payments'))
+        total_usd = get_quantized(sum(float(payment.get('paidUsdAmount', 0))
```

Comment:
we have to use get_quanitized, because floats are used throughout the code.. when you do any math 

43.39 * 4 for example.. you don't get a clean = 173.56

Instead with floats you can get 173.5599999999999999999999999999999999.

Which cause issues when seeing if there is sufficient funds.